extensions/amp-story/0.1/media-pool.js

```diff
@@ -457,41 +554,54 @@ export class MediaPool {
    * @private
    */
   swapPoolMediaElementIntoDom_(domMediaEl, poolMediaEl, sources) {
-    const ampMediaForPoolEl = ampMediaElementFor(poolMediaEl);
-    const ampMediaForDomEl = ampMediaElementFor(domMediaEl);
     poolMediaEl[REPLACED_MEDIA_PROPERTY_NAME] = domMediaEl.id;
 
     return this.enqueueMediaElementTask_(poolMediaEl,
         new SwapIntoDomTask(domMediaEl, this.vsync_))
         .then(() => {
-          this.maybeResetAmpMedia_(ampMediaForPoolEl);
-          this.maybeResetAmpMedia_(ampMediaForDomEl);
           this.enqueueMediaElementTask_(poolMediaEl,
               new UpdateSourcesTask(sources, this.vsync_));
           this.enqueueMediaElementTask_(poolMediaEl, new LoadTask());
+
+          this.triggerAllocationEvent_(poolMediaEl, /* isAllocated */ true);
```

Comment:
Technically, allocating a media element from the pool and actually inserting that media element into the DOM are different things; is there a reason why this is here instead of `allocateMediaElement_`?

Reply:
Sounds good.  Agreed the naming doesn't matter from the client perspective, it's just slightly confusing internal to `MediaPool`.  That can probably be mitigated with just a comment where the event is defined and/or triggered.